NuGet-UnitTests.slnf

```diff
@@ -50,7 +50,7 @@
       "test\\NuGet.Clients.Tests\\NuGet.VisualStudio.Common.Test\\NuGet.VisualStudio.Common.Test.csproj",
       "test\\NuGet.Clients.Tests\\NuGet.VisualStudio.Implementation.Test\\NuGet.VisualStudio.Implementation.Test.csproj",
       "test\\NuGet.Clients.Tests\\NuGetConsole.Host.PowerShell.Test\\NuGetConsole.Host.PowerShell.Test.csproj",
-      "test\\NuGet.Core.Tests\\Microsoft.Build.NuGetSdkResolver.Tests\\Microsoft.Build.NuGetSdkResolver.Test.csproj",
```

Comment:
Given that *most* of the test assembly have a singular `Test` instead of the plural `Tests`, my vote would be to align in that direction. 

What do you think @NuGet/nuget-client ?

Reply:
It's not broken, but changing it  now might break something. I want to keep `Test`.

Reply:
Majority doesn't mean much if the majority is wrong. If plural is the goal, we should move that direction.

Reply:
dotnet SDK names all of theirs plural: https://github.com/dotnet/sdk/tree/master/src/Tests

Reply:
> Majority doesn't mean much if the majority is wrong

I have seen both patterns, and I'm not sure you can argue one or the other is wrong.

With things as little as this, consistency is best. And moving to `.Test` named project is the path of least resistance at this point.

Reply:
There are some test projects named `.Tests` too. Either way, it should be consistent across the repo. I'd move to plural too! seeing as `dotnet/sdk` and its sister repos have the same naming as well.

Reply:
@Nirmal4G as a compromise with @nkolev92 , can you go ahead and move to .Test for now, then we can move *all* to plural later?

Reply:
Okay. But what about other projects that are already plural?

Also, isn't it kind of redundant to rename the project twice? i.e. from `Tests` to `Test` to `Tests`.

Reply:
I want to reiterate path of least resistance. 
There's no tangible benefit of changing everything (test folders in particular) around beyond a painful few months when we're developing on branch that might or might not have this change.

Reply:
I want to rephrase my recommendation to make sure we're on the same page. 

I am fully in support of aligning the naming for the NuGetSdkResolver test project. 
`.Test` or `.Tests` is stylistic, I am comfortable with either change for this particular project. 

I'd prefer to *not* change projects that are already aligned in project folder/project name. I think changing *all* test project and folder names introduces unnecessary churn for no clear benefit.